components/bl-pagination-component/src/helpers.js

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+
+export const useControlButtonClasses = (isDisabled, variant, paginationSize) => useMemo(() => {
+  const classes = ['pagination__control-button', variant];
+
+  if (isDisabled) {
+    classes.push('pagination__control-button--disabled');
+  }
+
+  if (paginationSize) {
+    classes.push(`${ variant }--${ paginationSize }`);
+  }
+
+  return classes.join(' ');
+}, [isDisabled, variant, paginationSize]);
+
+export const usePageListClasses = (currentPage, page, variant, paginationSize) => useMemo(() => {
```

Comment:
the same here

or, perhaps we can move this into particular components since it takes a couple lines